src/command.js

```diff
@@ -185,13 +188,17 @@ export default class StorybookCommand extends Command {
       this.preview = { content: previewDOM, sha: sha256hash(previewDOM) };
       clearTimeout(logTimeout);
 
-      // borrow a browser page to discover stories
-      await this.percy.browser.launch();
       page = await this.percy.browser.page({ meta });
       await page.goto(previewUrl);
 
       /* istanbul ignore next: no instrumenting injected code */
-      return await page.eval((util, previewUrl) => {
+      return await page.eval(async ({ waitFor }, previewUrl) => {
+        // ensure the page has loaded and the var we need is present
+        await waitFor(() => !!window.__STORYBOOK_CLIENT_API__, 5000)
+          .catch(() => Promise.reject(new Error(
+            'Storybook object not found on the window. ' +
+            'Open Storybook and check the console for errors.'
+          )));
```

Comment:
Now that we have this check & error, we don't need to do the same check & error below.

Also, small style thing, can we add a newline below this while your making the above change?